### PR TITLE
Add Tjuka-2022-220

### PR DESCRIPTION
**Pull request by AnnikaTjuka**
# Pull request checklist

- [X] add new concept list
- [ ] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

This PR adds an extended version of Tjuka-2021-192.

### CODE REVIEW

File: concepticondata/references/references.bib [hunk 1]
```diff
@@ -1,3 +1,11 @@
+@Misc{Tjuka2022,
```
Comment (LinguList) [inline concepticondata/references/references.bib:1]: We quote our blog posts now as a journal. So you should write @article instead.

Like here:

```
@Article{Tjuka2020TBLOG1,
  author    = {Tjuka, Annika},
  title     = {Adding concept lists to Concepticon: A guide for beginners},
  journal   = {Computer-Assisted Language Comparison in Practice},
  year      = {2020},
  number    = {1},
  url       = {https://calc.hypotheses.org/2225},
  volume    = {3},
}
```

This year is volume 5, and page range is the range of the pages in your PDF.

Comment (LinguList) [reply concepticondata/references/references.bib:1]: So I'd recommend to not quote Zenodo, which is referenced in the blogpost, but rather the article/blog post referencing this list. The URL / DOI can still be added from Zenodo.

**Review by LinguList — APPROVED** [review]
I already approve, but recommend to addjust the reference, as I suggested.

**Comment by AnnikaTjuka** [comment]
I updated the reference for the current list and also for the previous version.

**Review by LinguList — APPROVED** [review]

**Pull request merged**